### PR TITLE
Add cross Icon to test

### PR DESCRIPTION
**Pull request by rnukala1982**
To create Cross Icon
Screenshot 2024-03-16 at 5 49 17 PM

### CODE REVIEW

**Comment by vood** [comment]
This is great first start! Thank you. Would you mind to do small visual tweaks before merge. Currently the added X creates a lot of empty space on the right side, which is problematic on mobile screens where vertical space is very precious, to fix it put X inline with the Yearly/Monthly ToggleGroup. You can achieve this either using position absolute with top-X, right-X tailwind classes or by inserting X button in the same parent div with ToggleGroup and working with flex, justify-between tailwind classes. 

Thanks again for your work.

**Comment by rnukala1982** [comment]
Addressed the feedback. The ui looks as below post changes:

Screenshot 2024-03-17 at 10 17 58 AM

**Comment by rnukala1982** [comment]
Updated the PR to fix this. Thanks for catching this. Hope fully this matches the expectation.

File: app/[locale]/setup/page.tsx [hunk 1]
```diff
@@ -210,7 +211,7 @@ export default function SetupPage() {
             showNextButton={true}
             showBackButton={true}
           >
-            <Plans />
```
Comment (vood) [inline app/[locale]/setup/page.tsx:213]: I hate to do this, and it is my fault, I overlooked this nuance originally, but hey, an opportunity to learn the codebase :) 

So the `Plans` component is reusable and is shown in two places. One during the onboarding in `setup/page.tsx`, and the second time within the app itself in `plan-picker.tsx`.

We only need a cross icon in `plan-picker.tsx` and we do not need it in `setup/page.tsx`

You can achieve this two ways:
1. Introduce a `showCloseIcon` argument in Plans and making `onClose` optional
2. Moving the close icon code to plan-picker, this way you won't need to change `Plans` component at all

Thoughts?

Below the X showing in the onboarding, just for the reference (we don't need it there as I said earlier). 

Screenshot 2024-03-17 at 4 00 49 PM

Comment (rnukala1982) [reply app/[locale]/setup/page.tsx:213]: Sure will check the options and fix it. I prefer option1 so if we need to reuse it anywhere we dont have to comeback and code again. Just flip the flag. I will check on changes in a little bit later today.

Comment (rnukala1982) [reply app/[locale]/setup/page.tsx:213]: Added the changes.

**Review by vood — APPROVED** [review]

**Pull request merged**